test/parallel/test-cluster-disconnect-exitedAfterDisconnect-exit.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const common = require('../common');
+
+// Test should fail in Node.js 5.4.1 and pass in later versions.
```

Comment:
I think it's probably safe to drop this comment.

Reply:
The comment is probably incorrect as it was likely copied from test/parallel/test-cluster-disconnect-exitedAfterDisconnect-race.js (where it is, in fact, correct).

Reply:
Fixed